build.gradle

```diff
@@ -5,10 +5,10 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
Does it build with these deps for you?  I know we had an issue with `2.1.3` in the past.